packages/tpetra/core/test/FECrsMatrix/FECrsMatrix_UnitTests.cpp

```diff
@@ -301,15 +302,97 @@ TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL( FECrsMatrix, Assemble1D, LO, GO, Scalar, Node
   TPETRA_GLOBAL_SUCCESS_CHECK(out,comm,success)
 }
 
-////
-TEUCHOS_UNIT_TEST_TEMPLATE_4_DECL( FECrsMatrix, Assemble1D_Kokkos, LO, GO, Scalar, Node )
-{
+//// specializations for kokkos fecrsmatrix assembly
+//std::complex<double>
+template<class LO, class GO, class Scalar, class Node>  
```

Comment:
1. This code needs to be protected by the `HAVE_TPETRA_INST_COMPLEX_DOUBLE` macro, since `Scalar=std::complex<double>` can be enabled or disabled at configure time.

2. It's not clear to me why this specialization needs to exist, since there's nothing in it that depends on Scalar being `std::complex<double>`.  (See note below about `Kokkos::complex<double>`.)